<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddHistory</class>
 <widget class="QWidget" name="AddHistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加记录</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/resource.qrc">
    <normaloff>:/icon/main.ico</normaloff>:/icon/main.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QFormLayout" name="formLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>财富值</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>支付类型</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="recordOut">
           <property name="text">
            <string>支出</string>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>20</height>
            </size>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="recordIn">
           <property name="text">
            <string>收入</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>类型</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="type_select"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>日期</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCalendarWidget" name="moneyCalendar"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>备注</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPlainTextEdit" name="remarkTextEdit"/>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="commitBtn">
         <property name="text">
          <string>提交</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="returnMsg">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="1">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="money_input">
           <property name="text">
            <string>00</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" alignment="Qt::AlignLeft">
          <widget class="QLineEdit" name="money_yuan_input">
           <property name="text">
            <string>10</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
